src/fsharp/MethodCalls.fs

```diff
@@ -445,7 +514,7 @@ type CalledMeth<'T>
 
     member x.ParamArrayCallerArgs = x.ArgSets |> List.tryPick (fun argSet -> if Option.isSome argSet.ParamArrayCalledArgOpt then Some argSet.ParamArrayCallerArgs else None )
 
-    member x.ParamArrayElementType = 
+    member x.GetParamArrayElementType() = 
```

Comment:
Curious if you changed that to a method due to the assert showing in variable inspector during debug or if it is more stylistic about the thing not being a property?

Reply:
> Curious if you changed that to a method due to the assert showing in variable inspector during debug or if it is more stylistic about the thing not being a property?

For the former reason